src/sentry/snuba/metrics_layer/query.py

```diff
@@ -584,19 +586,34 @@ def _query_meta_table(
     if extra_condition:
         conditions.append(extra_condition)
 
-    counters_query = (
-        Query(Entity("generic_metrics_counters_meta"))
-        .set_select([Column("project_id"), Column(column_name)])
-        .set_groupby([Column("project_id"), Column(column_name)])
-        .set_where(conditions)
-        .set_orderby(
-            [
-                OrderBy(Column("project_id"), Direction.ASC),
-                OrderBy(Column(column_name), Direction.ASC),
-            ]
+    if in_random_rollout("sentry-metrics.metrics-layer.use-storage-direct-meta-queries"):
```

Comment:
where you able to run the CI tests using the storage-direct-meta-queries on?

Reply:
Tested on CI with flag enabled 👍